### PR TITLE
Match Node docs prettier options to sdk-node options

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

**Comment by lorensr** [comment]
GitHub action error: `git fetch` exiting with error code. Have gotten this a couple times now, seems intermittent.

image 

Normal output:

image

**Review by flossypurse — APPROVED** [review]

File: src/css/custom.css [hunk 1]
```diff
@@ -210,51 +210,51 @@ h4,
 }
 
 .related-read-archetype-tag {
-  padding:3px 6px;
+  padding: 3px 6px;
```
Comment (vkarpov15) [inline src/css/custom.css:213]: Is prettier modifying CSS? Strange.

File: .prettierrc.json [hunk 2]
```diff
@@ -14,5 +14,14 @@
   "singleQuote": false,
   "tabWidth": 2,
   "trailingComma": "es5",
-  "useTabs": false
+  "useTabs": false,
+  "overrides": [
```
Comment (vkarpov15) [inline .prettierrc.json:18]: Nice, didn't know you could do this :+1:

**Review by vkarpov15 — APPROVED** [review]

File: src/css/custom.css (hunk 1 above)
Comment (lorensr) [reply src/css/custom.css:213]: Yep, that's just what happened when I ran `npm run format`

**Comment by bergundy** [comment]
Merged this despite the prettier failure for obvious reasons.

**Pull request merged**